rust-tests/src/lib.rs

```diff
@@ -64,6 +64,39 @@ mod tests {
         assert_eq!(pubkey_set.len(), 5); // Ensure keys at various derivation paths are unique
     }
 
+    /// This test requires interactive approval of message signing on the ledger.
+    #[test]
+    #[serial]
+    fn ledger_sign_garbage_test() {
+        let wallet_manager = initialize_wallet_manager();
```

Comment:
@CriesofCarrots, the existing code set a precedent for copy-pasta.

Reply:
I'll rework that in a helper fn at some point.